lib/apartment/active_record/schema_migration.rb

```diff
@@ -1,11 +1,23 @@
 # frozen_string_literal: true
+#
+require 'active_record/schema_migration'
 
 module ActiveRecord
-  class SchemaMigration < ActiveRecord::Base # :nodoc:
-    class << self
-      def table_exists?
-        connection.table_exists?(table_name)
+  if ActiveRecord::SchemaMigration < ActiveRecord::Base
```

Comment:
suggestion: I think this condition is unneccessary, as opening an existing class without the parent class name doesn't change the inheritance, so this:
```
class SchemaMigration
```
should work no matter the parent.